crates/sui-transactional-test-runner/src/test_adapter.rs

```diff
@@ -1854,6 +1868,7 @@ async fn init_sim_executor(
     reference_gas_price: Option<u64>,
     object_snapshot_min_checkpoint_lag: Option<usize>,
     object_snapshot_max_checkpoint_lag: Option<usize>,
+    db_name: String,
```

Comment:
Can we tweak this so that this function accepts a DB name salt, and it is responsible for generating the DB name and port number etc? Seems like it would be cleaner than trying to recover a number from the string?